src/main.rs

```diff
@@ -1,3 +1,178 @@
-fn main() {
-    println!("Hello, World!");
+/*
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ * SPDX-FileCopyrightText: Copyright 2023 KUNBUS GmbH
```

Comment:
```suggestion
 * SPDX-License-Identifier: GPL-2.0-or-later
 * SPDX-FileCopyrightText: Copyright 2023 KUNBUS GmbH
```
Just to be consistent with the other files